app/models/red_line_boundary_change_validation_request.rb

```diff
@@ -19,6 +16,28 @@ def update_planning_application!(params)
     planning_application.update!(boundary_geojson: new_geojson)
   end
 
+  def new_geojson=(value)
+    if value.is_a?(String)
+      return if value.blank?
+      value.gsub!(/(?<!\\)"=>nil/, '":null')
+      value.gsub!(/(?<!\\)"=>/, '":')
```

Comment:
These shouldn't be necessary - we shouldn't have to deal with hash strings, just json strings.